doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -395,7 +395,9 @@ Binary libraries
 Bluetooth libraries and services
 --------------------------------
 
-|no_changes_yet_note|
+* :ref:`bt_mesh` library:
+
+  * Fixed an issue where ``bt_mesh_adv_unref()`` function could assert when messaging to a proxy node.
```

Comment:
```suggestion
  * Fixed an issue where :c:func:`bt_mesh_adv_unref` function could assert when messaging to a proxy node.
```

Reply:
It won't work as this is not a public API.

Reply:
But customers figured out that assertion happens in this function (dev zone case for the reference: https://devzone.nordicsemi.com/f/nordic-q-a/117593/assertion-fail-west_topdir-zephyr-subsys-bluetooth-mesh-adv-c). That is why I think it is important to write function name here.

Reply:
Okay, then leave it with double backticks, but add *the*.

Reply:
Added!